src/raftstore/store/store.rs

```diff
@@ -1217,6 +1228,11 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                 ExecResult::VerifyHash { index, hash } => {
                     self.on_ready_verify_hash(region_id, index, hash)
                 }
+                ExecResult::CompactRanges { ranges } => {
+                    for range in ranges {
```

Comment:
I worry this may cause many manual compactions which may reduce the performance.